birdhouse/config/jupyterhub/jupyterhub_config.py.template

```diff
@@ -86,7 +101,7 @@ c.Spawner.disable_user_config = True
 c.DockerSpawner.default_url = '/lab'
 c.DockerSpawner.remove = True  # delete containers when servers are stopped
 ${ENABLE_JUPYTERHUB_MULTI_NOTEBOOKS}    # noqa
-c.DockerSpawner.pull_policy = "always"  # for images not using pinned version
+# c.DockerSpawner.pull_policy = "always"  # for images not using pinned version
```

Comment:
Is there a reason to comment out this line?

This is there for the case when user use "latest" flavor of image so new update will not be detected.